src/main/java/nl/knaw/dans/filemigration/api/EasyFile.java

```diff
@@ -0,0 +1,125 @@
+package nl.knaw.dans.filemigration.api;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "easy_files")
+@NamedQueries({ @NamedQuery(name = "EasyFile.findByDatasetId",
+                            query = "SELECT * FROM EasyFile ef WHERE ef.dataset_sid = :datasedSid ORDER BY path"),
```

Comment:
Here are some examples of NamedQueries and using parameters:
[LicenseServiceBean(or LicenseDAO in Dropwizard)](https://github.com/DANS-KNAW/dataverse/blob/a79143e6b61047e1b61e28b671f03b90bfdc405e/src/main/java/edu/harvard/iq/dataverse/LicenseServiceBean.java#L55-L65)

[License(Entity class)](https://github.com/DANS-KNAW/dataverse/blob/a79143e6b61047e1b61e28b671f03b90bfdc405e/src/main/java/edu/harvard/iq/dataverse/License.java#L21-L39)